modules/architecture-machine-roles.adoc

```diff
@@ -8,7 +8,9 @@
 
 [NOTE]
 ====
-The cluster also contains the definition for the bootstrap role. Because the bootstrap machine is used only during cluster installation, its function is explained in the cluster installation documentation.
+* The cluster also contains the definition for the bootstrap role. Because the bootstrap machine is used only during cluster installation, its function is explained in the cluster installation documentation.
+
+* The {product-title} version must match between control plane host and node host. For example, in a 4.9 cluster, all control plane hosts must be 4.9 and all nodes must be 4.9. In some cases, temporary mismatches during cluster upgrades might arise. For odd {product-title} minor versions, kubelet versions 0 or 1 minor versions behind the API server version are supported. For even {product-title} minor versions, kubelet versions 0, 1, or 2 minor versions behind the API server version are supported.
```

Comment:
I wish we could define 'some cases' better. Is is possible to enumerate them? Is this for upgrades? Disaster Recover? How long can my cluster be in this state? What are the boundaries of support? 

IMO this wording is to ambgous and needs to be refined better, using technical limitations and boundaries. 

If we don't do this; we might as well just say we support version drift (as you have stated it for odd and even minor versions - as confusing as that is).